WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,15 +38,15 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v20.0-whats-new"
+msgctxt "v20.1-whats-new"
 msgid ""
-"We’re kicking off version 20 with a fresh new look. The Quick Start card and Next Steps tours now match the new tour designs for existing sites, including a snazzy illustration. High five.\n"
+"Need a little writing inspiration? We have you covered with brand-new blogging prompts. Get a new prompt each day, write a post about it, and build a writing habit in no time.\n"
```

Comment:
The original copy used "inspo." I replaced it with the more formal "inspiration" to simplify the translation process.

<img width="574" alt="image" src="https://user-images.githubusercontent.com/1218433/173978669-4339c521-b253-458d-b6b4-334ffb23d1f9.png">

If you feel strongly this wasn't the right call, feel free to revert and run `bundle exec fastlane update_appstore_strings version:20.1` one more time.

Reply:
I was also surprised by this word when I saw it in the copy, because as a non-native speaker I wasn't familiar with this informal word in English, and while I guessed/figured what it would mean, I feel that using the more formal "inspiration" is better 👍